Documentation/ApiOverview/Routing/Introduction.rst

```diff
@@ -48,12 +50,15 @@ Slug
     * A slug only contains characters which are allowed within URLs. Spaces, commas and other special characters are converted to a fallback character.
     * A slug is always lower-cased.
     * A slug is unicode-aware.
+    * Slugs must be separated by one or more character like "/", "-", "_" and "&".
+      Normal characters should not be used as separators for better readability.
 
 .. note::
 
     Until TYPO3 v9 routing for TYPO3 was done by using extensions such as `realURL` or `coolURI`.
-    In contrast to concepts within RealURL of "URL segments", a slug is a segment of a URL, but it does not have
-    to be separated by slashes. Therefore, a slug can contain slashes.
+    In contrast to concepts within RealURL of "URL segments", a slug is a segment of a URL.
+    Slugs should be separated by slashes, but they do not have
+    to. Therefore, a slug of a record can contain slashes.
```

Comment:
```suggestion
    Slugs should be separated by slashes, but this is not a strict requirement. Therefore, a slug
    of a record may contain slashes. However, the risk of conflicts are higher when having slugs
    with slashes since unrelated page hierarchies can have the same slug.
```